apps/site/lib/site_web/controllers/cms_controller.ex

```diff
@@ -43,6 +45,22 @@ defmodule SiteWeb.CMSController do
     |> handle_page_response(conn)
   end
 
+  def reset_cache_key(conn, %{"object" => object, "id" => id}) do
+    @cache.delete("/cms/#{object}/#{id}")
```

Comment:
Does that status of these requests need to be checked?  It seems this could open silent errors if the cache request failed, but we logged it and returned it as a success anyway

Reply:
No, the reason we're returning a 202 is to indicate that we accepted the request. We aren't indicating whether or not it was a success (200).